main.go

```diff
@@ -136,6 +136,14 @@ func (p Ploop) Init() (*flexvolume.Response, error) {
 	}, nil
 }
 
+func (p Ploop) realWorkingDir(options map[string]string) string {
+	realDir := workingDir
+	if options["secretFromSystem"] == "true" {
```

Comment:
workingDir is not a secret, thus it is not obvious that it is managed by 'secretFromSystem'. Let's name this option something like 'optionsFromSystem'.